config/arc_cluster/config.yaml

```diff
@@ -0,0 +1,35 @@
+cluster:
+  mkdir -p logs/{rule} &&
+  sbatch
+    --partition={resources.partition}
+    --qos={resources.qos}
+    --cpus-per-task={threads}
+    --mem={resources.mem_mb}
+    --job-name=smk-{rule}-{wildcards}
+    --output=logs/{rule}/{rule}-{wildcards}-%j.out
+    --export=ALL
+    --parsable
+default-resources:
+  - qos=standard  # {basic, standard, priority} only have credits for standard
+  - partition=short  # {short, medium, long, devel, interactive}
+  - mem_mb=16000
+  - time="01:00:00"
+restart-times: 1  # if a job fails, retry it once
+max-jobs-per-second: 16  # maximum jobs to _submit_ per second
+max-status-checks-per-second: 1
+local-cores: 1
+latency-wait: 3  # seconds to wait for files to appear before failing
```

Comment:
Should this be a little higher - wait 10 or 30s for filesystem?

Reply:
Yeah I've already run into this a few times on 3s -- I thought the Infiniband would cope! I'll raise it to 20s.